depthai_nodes/ml/parsers/yolo.py

```diff
@@ -331,7 +333,7 @@ def run(self):
                     results[i, 6:],
                 )
 
-                bbox = xyxy2xywhn(bbox, input_shape)
+                bbox = xyxy_to_xywh_norm(bbox, input_shape)
```

Comment:
Rather use `xyxy_to_xywh(bbox)` and then `normalize_bboxes(bboxes, height, width` so there is no need to make new function in bbox_format_converters.

Reply:
Hi @aljazkonec1 thanks for the comment.
This would not work in this case since we pass a single bbox to the `xyxy_to_xywh_norm` and not a batch of bboxes (shape (4,) instead of (N, 4)).
I could either add an extra dim to the `bbox` before passing to the `xyxy_to_xywh(bbox)` and then `normalize_bboxes` or use this new `xyxy_to_xywh_norm` which is for a single bbox.

Reply:
Is it a problem if we normalize all bboxes after the for loop? I see you use the bbox again in line 351 but the  function aexpects int type bbox, which I'm guessing is it the xyxy format.

Can you check if normalization after for loop and parsing un-normalized bbox to `process_single_mask` causes an error? And if it doesn't I would move normalization out of the loop.

Reply:
I just updated the code and now we are using the `xyxy_to_xywh` and `normalize_bboxes`. Also I removed completely the `xyxy_to_xywh_norm` function.
The normalization needs to be done in the loop since the `process_single_mask` requires the bbox to be normalized to correctly do the cropping of the mask to the respective bbox.
I believe this workaround (as pushed in the latest commit) should be fine. What do you think @aljazkonec1?

Reply:
Yes if `process_single_mask` needs normalized bbox than this is fine fix. Eventually we would like to also replace `process_single_mask` with just `process_masks` but that will come at a later PR. Looks good!